app/assets/stylesheets/components/_auction-show.scss

```diff
@@ -121,15 +120,6 @@
   margin-bottom: $base-padding-lite;
   margin-top: 1rem;
 
-  a {
-    text-decoration: underline;
-
-    &:hover,
-    &:focus {
-      text-decoration: none;
```

Comment:
Just curious, is there a reason we are having the links drop the underlining when the user hovers or focuses? This doesn't seem to be in the USWDS and I worry it'll be a bit confusing to have links seemingly vanish when you visit them (I know the link is still there, but we took its decoration away)